src/SFA.DAS.ApprenticeFeedback.Jobs/Infrastructure/Api/IApprenticeFeedbackApi.cs

```diff
@@ -31,6 +31,6 @@ public interface IApprenticeFeedbackApi
         Task<UpdateApprenticeFeedbackTargetResponse> UpdateFeedbackTarget([Body] UpdateApprenticeFeedbackTargetRequest apprenticeFeedbackTargetToUpdate);
 
         [Post("feedbacktransaction")]
-        Task<GetEmailTransactionsResponse> GenerateEmailTransactions();
+        Task<NullResponse> GenerateEmailTransactions();
```

Comment:
Why does this need to be an empty class (`NullResponse`), could you use `null` instead or have a `Task` they way it is for `GenerateFeedbackSummaries();`

Reply:
Removed the NullResponse it is needed in the Outer API but not needed to be passed to the jobs